src/server/common/helpers/proxy/proxy-agent.js

```diff
@@ -1,33 +0,0 @@
-import { URL } from 'node:url'
-import { HttpsProxyAgent } from 'https-proxy-agent'
```

Comment:
Looks like the `https-proxy-agent` dependency needs uninstalling if its not used anywhere else?
`npm uninstall https-proxy-agent`